enterprise/functions/src/main/java/io/crate/operation/aggregation/HyperLogLogDistinctAggregation.java

```diff
@@ -88,7 +88,7 @@ private HyperLogLogDistinctAggregation(FunctionInfo info, DataType dataType) {
     public HllState newState(RamAccountingContext ramAccountingContext,
                              Version indexVersionCreated,
                              BigArrays bigArrays) {
```

Comment:
I think we could remove the BigArrays argument now from the interface method as only HLL used it. Also this would avoid that later on someone is using this BigArrays instance which uses a circuit breaker and thus accounts..

Reply:
I'll do that in a separate PR without backport